packages/calcite-design-tokens/CHANGELOG.md

```diff
@@ -3,136 +3,24 @@
-
-**Note:** Version bump only for package @esri/calcite-design-tokens
-
-## [3.0.0-next.2](https://github.com/Esri/calcite-design-system/compare/@esri/calcite-design-tokens@3.0.0-next.1...@esri/calcite-design-tokens@3.0.0-next.2) (2024-12-02)
-
-**Note:** Version bump only for package @esri/calcite-design-tokens
-
-## [3.0.0-next.1](https://github.com/Esri/calcite-design-system/compare/@esri/calcite-design-tokens@3.0.0-next.0...@esri/calcite-design-tokens@3.0.0-next.1) (2024-12-02)
-
-**Note:** Version bump only for package @esri/calcite-design-tokens
-
-## [3.0.0-next.0](https://github.com/Esri/calcite-design-system/compare/@esri/calcite-design-tokens@2.3.0-next.4...@esri/calcite-design-tokens@3.0.0-next.0) (2024-11-27)
+## [3.0.0](https://github.com/Esri/calcite-design-system/compare/@esri/calcite-design-tokens@2.2.0...@esri/calcite-design-tokens@3.0.0) (2025-02-08)
 
 ### ⚠ BREAKING CHANGES
 
-- **tokens:** Size and Space token values have changed. Line Height
-  tokens exported for JS and ES6 are now unitless.
-
-### Size
-
-calcite-size-px: 1px;
-calcite-size-xxxs: 0.75rem; // 12px was 2px
-calcite-size-xxs: 0.875rem; // 14px was 4px
-calcite-size-xs: 1rem; // 16px was 6px
-calcite-size-sm: 1.5rem; // 24px was 8px
-calcite-size-sm-plus; // removed
-calcite-size-md: 2rem; // 32px was 12px
-calcite-size-md-plus; // removed
-calcite-size-lg: 2.75rem; // 44px was 16px
-calcite-size-xl: 3rem; // 48px was 20px
-calcite-size-xxl: 4rem; // 64px was 24px
-calcite-size-xxxl: 6rem; // 96px was 32px
-
-### Space
-
-calcite-spacing-none: 0; // new
-calcite-spacing-px: 1px;
-calcite-spacing-base: 2px;
-calcite-spacing-xxs: 0.25rem; // 4px
-calcite-spacing-xs: 0.375rem; // 6px
-calcite-spacing-sm: 0.5rem; // 8px
-calcite-spacing-sm-plus: 0.625rem; // new
-calcite-spacing-md: 0.75rem; // 12px
-calcite-spacing-md-plus: 0.875rem; // new
-calcite-spacing-lg: 1rem; // 16px was 14px
-calcite-spacing-xl: 1.25rem; // 20px was 16px
-calcite-spacing-xxl: 1.5rem; // 24px was 20px
-calcite-spacing-xxxl: 2rem; // 32px
-
-### Pixel based tokens are now categorized under "fixed"
-
-calcite-size-fixed-xxxl: 32px; // deprecated
-calcite-size-fixed-xxl: 24px; // deprecated
-calcite-size-fixed-xl: 20px; // deprecated
-calcite-size-fixed-lg: 16px; // deprecated
-calcite-size-fixed-md: 12px; // deprecated
-calcite-size-fixed-sm: 8px; // deprecated
-calcite-size-fixed-xs: 6px; // deprecated
-calcite-size-fixed-xxs: 4px; // deprecated
-calcite-size-fixed-xxxs: 2px; // deprecated
-
-calcite-spacing-fixed-lg: 14px; // deprecated
-calcite-spacing-fixed-md: 12px; // deprecated
-calcite-spacing-fixed-sm: 8px; // deprecated
-calcite-spacing-fixed-xl: 16px; // deprecated
-calcite-spacing-fixed-xs: 6px; // deprecated
-calcite-spacing-fixed-xxl: 20px; // deprecated
-calcite-spacing-fixed-xxs: 4px; // deprecated
-calcite-spacing-fixed-xxxl: 32px; // deprecated
```

Comment:
None of the sections from the tokens' `BREAKING CHANGES` to here ended up in the v3 changelog generated by release-please. If they should be there, please create a follow-up PR. cc @alisonailea @geospatialem @jcfranco